src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -11,14 +11,36 @@ mon_ip=$(ifconfig eth0  | grep 'inet ' | awk '{ print $2}')
 cephadm bootstrap --mon-ip $mon_ip --initial-dashboard-password {{ admin_password }} --allow-fqdn-hostname --dashboard-password-noupdate --shared_ceph_folder /mnt/{{ ceph_dev_folder }}
 
 fsid=$(cat /etc/ceph/ceph.conf | grep fsid | awk '{ print $3}')
+export='export'
+ceph_cmd="cephadm shell --fsid ${fsid} -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring"
 
 {% for number in range(1, nodes) %}
   ssh-copy-id -f -i /etc/ceph/ceph.pub  -o StrictHostKeyChecking=no root@{{ prefix }}-node-0{{ number }}.{{ domain }}
   {% if expanded_cluster is defined %}
-    cephadm shell --fsid $fsid -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring ceph orch host add {{ prefix }}-node-0{{ number }}.{{ domain }}
+    ${ceph_cmd} ceph orch host add {{ prefix }}-node-0{{ number }}.{{ domain }}
   {% endif %}
 {% endfor %}
 
 {% if expanded_cluster is defined %}
-  cephadm shell --fsid $fsid -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring ceph orch apply osd --all-available-devices
+  ${ceph_cmd} ceph orch apply osd --all-available-devices
+{% endif %}
+
+{% if nfs is defined %}
+  sleep 30 # waiting for OSDs to get in/up
+
+  # add metadata pool
+  ${ceph_cmd} ceph osd pool create mp
+  # add data pool
+  ${ceph_cmd} ceph osd pool create dp
+
+  # apply mds service
+  ${ceph_cmd} ceph orch apply mds mymds ceph-node-00.cephlab.com
+  # add mds daemon
+  ${ceph_cmd} ceph orch daemon add mds mymds ceph-node-00.cephlab.com
+
+  # create filesystem
+  ${ceph_cmd} ceph fs new myfs mp dp
```

Comment:
@avanthakkar AFAIK this is the old way to delpoy a filesystem, as we now have the simpler:
```ceph fs volume create <my-fs>```.
Also, taking into account that this is for E2E testing purposes, this should be done using the GUI (cluster > Services) so we increase our coverage. If it's too complicated to do it in this PR, let's do it in a follow-up PR.